doc/user_guide/utilities.rst

```diff
@@ -96,3 +96,36 @@ wavelength and pixel position in the detector plane) follows the conventions
 described in the tutorial from  `Horiba Scientific
 <https://horiba.com/uk/scientific/products/optics-tutorial/wavelength-pixel-position>`_.
 
+
+
+Cropping multiple signals in the navigation axis 
+================================================
+
+The function :func:`~.utils.signals.crop_edges`
+removes the specified number of pixels or % from the four edges of a spectral map,
+from the edges inwards. It takes a list of `Signals` and cropping can happen
+uniformly on all sides or by specifying the cropping range for each axis or each
+side. If the navigation axes shape across the list of signals is different, all
+signals can be rebinned to match the shape of the first signal in the list.
+It is a convenience wrapper for the :external+hyperspy:ref:`method in HyperSpy <signal.indexing>`.
+The function can also be called as s.crop_edges() on a single signal object.
+
+.. code-block:: python
+
+    >>> signals = [cl_map, sem_image]
+    >>> signals
+    [CLSpectrum <256,256|1024>, Signal2D <128,128|1>]
+    >>> signals_cropped = lum.utils.crop_edges(signals, crop_range=5, crop_units="%", rebin_nav=True)
+    >>> signals_cropped
+    [CLSpectrum <243,243|1024>, Signal2D <243,243|1>]
+    
+.. Note::
+
+    Many scanning luminescence techniques result in edge defects at the edges of the scanned region.
+    This function enables the same cropping of the navigation axis for a list of signals in the same
+    region to correct for such defect.
+
+.. Note::
```

Comment:
This note is not needed any more, as the object-oriented functionality is kept and for the parameter deprecation the mentioning in the changelog and warnings will be enough.